index.html

```diff
@@ -4,12 +4,84 @@
 </head>
 <body>
+    <div id="mobile-popup">
+        <img id="close-icon" src="./images/icons/close.svg" alt="close-icon" />
+        <img src="./images/logos/logo_faiz_festival_image.png" alt="nav bar image" id="nav-bar-image">
+        <hr id="nav-bar-hr" class="solid-divider-line txt-c">
+    <ul id="nav-bar" class="navbar-list-mobile">
+    </ul>
+    </div>
     <div id="main">
-        Hello World
+        <div id="landing">
+            <div id="landing-child">
+                <div id="top-nav">
+                    <ul id="top-nav-items">
+                        <li id="google_translate_element"></li>
+                    </ul>
+                </div>
+                <div id="nav">
+                    <ul id="nav-items-mobile">
+                        <li id="hamburger"><i class="fas fa-bars"></i></li>
+                    </ul>
+                    <nav id="nav-items-desktop">
+                        <img id="nav-logo" src="./images/logos/logo_faiz_festival_text.png" alt="">
+                        <ul id="nav-items-list">
+                            <li class="nav-list-item"><a href="./index.html">Home</a></li>
+                            <li class="nav-list-item"><a href="./about.html">About</a></li>
+                        </ul>
+                    </nav>
+                </div>
+                <div id="landing-body-home">
+                    <p class="hello-text txt-l">"The Cultural Fest is here"</p>
+                    <h1 class="page-title txt-l">Let's Smile Again &#128512;<br> 6<sup>th</sup> Faiz Festival 2022</h1>
+                    <div class="landing-box ">
+                        The sixth Faiz Festival started with the collaboration of Jang Group of Newspapers at Alhamra Friday.  Faiz’s memorable photographs with friends and family were on exhibition at the Alhamra Art Gallery. The pictures on exhibit were all in sepia tone waiting for the fans to come.
+                    </div>
+                    <h2 id="timing">2022.03.04(Fri) ~ 07(Sun)</h2>
+                    <h2 id="venue">@Alhamra Halls, Mall Road, Lahore</h2>
+                </div>
+            </div>
+        </div>
+        <div id="events">
+            <h3 id="event-heading">Main Events</h3>
+            <hr class="solid-divider-line">
+            <ul id="events-list">
+            </ul>
+            <div id="join-event">Join Faiz Festival 2022</div>
+        </div>
+        <div id="speakers">
+            <h3 id="speaker-heading">Speakers</h3>
+            <hr class="solid-divider-line">
+            <ul id="speakers-list">
+            </ul>
+            <div id="speakers-more">
+                <p id="more-text">More</p>
+                <i id="more-icon" class="fas fa-solid fa-angle-down"></i>
+            </div>
+        </div>
+        <div id="partners">
+            <h3 id="partner-heading">Partners</h3>
+            <hr class="solid-divider-line">
+            <ul class="partners-list">
+            </ul>
```

Comment:
Your partner's section is facing the same issue with images 😥 Please make the necessary changes so that you meet the project requirements 😁

Reply:
Image Paths are updated now.